### PR TITLE
#957 Link underline should be the same as FG color

### PR DESCRIPTION
**Pull request by peacefullatom**
Fixes [#957](https://github.com/xtermjs/xterm.js/issues/957).

### CODE REVIEW

File: src/Linkifier.ts [hunk 1]
```diff
@@ -187,7 +188,7 @@ export class Linkifier extends EventEmitter implements ILinkifier {
     let text = this._terminal.buffer.translateBufferLineToString(absoluteRowIndex, false);
     let currentIndex = absoluteRowIndex + 1;
     while (currentIndex < this._terminal.buffer.lines.length &&
-        (<any>this._terminal.buffer.lines.get(currentIndex)).isWrapped) {
```
Comment (Tyriar) [inline src/Linkifier.ts]: This was intentionally like this as it makes it easier to see it's a wrapped line, not the first in the while loop.

Comment (peacefullatom) [reply src/Linkifier.ts]: Need some clarifications here.

If this check will be removed then `text` variable will contain all lines from visible part of  buffer.

This in turn will crash code.

So, I need to know what we trying to achieve by this code change.

Comment (Tyriar) [reply src/Linkifier.ts]: I just mean the 2 extra spaces that were removed 😃

Comment (peacefullatom) [reply src/Linkifier.ts]: Ok. I merged latest changes to this branch.

Ready to go! 😃

**Pull request merged**